src/ILVerify/src/ILImporter.Verify.cs

```diff
@@ -692,6 +702,7 @@ void StartImportingBasicBlock(BasicBlock basicBlock)
 
         void EndImportingBasicBlock(BasicBlock basicBlock)
         {
+            basicBlock.EndOffset = -2; // Mark as already verified
```

Comment:
It may be nice to introduce named constants for these, The magic offsets are starting to show up at too many places.

Reply:
Yes, I though so too. Do you prefer an enum defined inside BasicBlock or constant int values?

Reply:
I do not have a strong opinion either way. (constant int values may look slightly better - enums tend to require casts.)